src/api/users.ts

```diff
@@ -11,5 +11,7 @@ export function getCurrentUser() {
 }
 
 export function getUser(userId: string) {
```

Comment:
I don't think this gets called any more.

Reply:
Ok, I reverted that change. I'll file an issue to clean up the unused user calls.